agent/api/task/task.go

```diff
@@ -562,6 +586,83 @@ func (task *Task) initializeEFSVolumes(cfg *config.Config, dockerClient dockerap
 	return nil
 }
 
+func buildVolumeNameForExecAgentBinary(binaryName string) string {
+	return fmt.Sprintf("%s-%s", internalExecAgentNamePrefix, binaryName)
+}
+
+// initializeExecAgentResources specifies the necessary volumes and mount points in all of the task containers in order for the
+// exec agent to run upon container start.
+func (task *Task) initializeExecAgentResources(cfg *config.Config) error {
+	if !task.IsExecAgentEnabled() {
+		return nil
+	}
+
+	tId, err := task.GetID()
+	if err != nil {
+		return err
+	}
+
+	execAgentBinNames := []string{execAgentBinName, execAgentSessionWorkerBinName, execAgentSessionLoggerBinName}
+
+	// Append exec agent binaries to task volumes
+	for _, bn := range execAgentBinNames {
```

Comment:
can just call it 'bin' instead of 'bn'

Reply:
bn stands for binary name. I am following the style guide here https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
I would change the comment above -- as I read this top to bottom I see `// Append exec agent binaries to task volumes` then I see `for _, bn` which makes me read "binary"
maybe if you say `// for each binary name, append its binary to task volume` to make this read immediately as `bn == binary name`

Reply:
Fixed (I think :))